app-editors/retext/retext-7.2.0-r1.ebuild

```diff
@@ -28,17 +30,29 @@ fi
 
 LICENSE="GPL-2+"
 SLOT="0"
+# See https://bugs.gentoo.org/772197.
+IUSE="enchant +markdown test webengine"
 RESTRICT="!test? ( test )"
 
 RDEPEND="
 	dev-python/chardet[${PYTHON_USEDEP}]
 	dev-python/docutils[${PYTHON_USEDEP}]
-	dev-python/markdown[${PYTHON_USEDEP}]
 	dev-python/markups[${PYTHON_USEDEP}]
 	dev-python/pygments[${PYTHON_USEDEP}]
-	dev-python/python-markdown-math[${PYTHON_USEDEP}]
-	dev-python/PyQt5[gui,network,printsupport,widgets,${PYTHON_USEDEP}]
-	dev-python/PyQtWebEngine[${PYTHON_USEDEP}]
+	dev-python/PyQt5[dbus,gui,printsupport,widgets,${PYTHON_USEDEP}]
+	enchant? ( dev-python/pyenchant[${PYTHON_USEDEP}] )
+	markdown? (
+		dev-python/markdown[${PYTHON_USEDEP}]
+		dev-python/python-markdown-math[${PYTHON_USEDEP}]
+	)
+	test? (
+		dev-python/PyQt5[testlib,${PYTHON_USEDEP}]
+		x11-base/xorg-server[xvfb]
+	)
```

Comment:
Test deps should be in `BDEPEND` and the `x11-base/xorg-server[xvfb]` is handled via *virtualx.eclass* that needs it.

Reply:
fixed locally